src/cosalib/qemuvariants.py

```diff
@@ -102,6 +102,10 @@
     "nutanix": {
         "image_format": "qcow2",
         "platform": "nutanix",
+        "skip_compression": True,
+        "convert_options": {
+            '-c': None
```

Comment:
it doesn't seem to work either

Reply:
```
Traceback (most recent call last):
  File "/usr/lib/coreos-assembler/cmd-buildextend-nutanix", line 120, in <module>
    artifact_cli().build_artifacts()
  File "/usr/lib/coreos-assembler/cosalib/build.py", line 414, in build_artifacts
    self._build_artifacts(*args, **kwargs)
  File "/usr/lib/coreos-assembler/cosalib/qemuvariants.py", line 328, in _build_artifacts
    meta_patch = self.mutate_image()
  File "/usr/lib/coreos-assembler/cosalib/qemuvariants.py", line 259, in mutate_image
    run_verbose(cmd)
  File "/usr/lib/coreos-assembler/cosalib/cmdlib.py", line 55, in run_verbose
    print("+ {}".format(subprocess.list2cmdline(args)))
  File "/usr/lib64/python3.10/subprocess.py", line 561, in list2cmdline
    for arg in map(os.fsdecode, seq):
  File "/usr/lib64/python3.10/os.py", line 822, in fsdecode
    filename = fspath(filename)  # Does type-checking of `filename`.
TypeError: expected str, bytes or os.PathLike object, not NoneType
+ rc=1
+ set +x
```

Reply:
Modified the convert options usage to have explicit handling for nil value for flags that are not followed with a parameter.